dist/components/tabs/_tabs.scss

```diff
@@ -0,0 +1,133 @@
+// Tabs
+// ====
+//
+// Tabs component with tabs controls and tabs body.
+// Horizontal controls tabs items placed in one line.
+
+
+// Configurable Variables
+// ---
+
+$tabs__item-size: 12rem !default;
+$tabs__padding: 0.5rem 0.75rem !default;
+$tabs__body-space: if(variable-exists(v-space), $v-space, 10px) !default;
+$tabs__active-background: if(variable-exists(active-link-color), $active-link-color, #16a085) !default;
+
+
+// Tabs
+// ---
+//
+
+.c-tabs {
+
+}
+
+
+// Tabs: Controls
+// ---
+//
+// List of tabs buttons
+//
+// Notes:
+//
+// 1. Add overflow scroll to avoid squished tabs
+// 2. Smooth scrolling on device
+// 3. Needed for correctly calculating the position to scroll to when the user has selected a tab
+//
+
+.c-tabs__controls {
+    position: relative; // 3
+
+    display: flex;
+    overflow-y: scroll; // 1
+    -webkit-overflow-scrolling: touch; // 2
+}
+
+
+// Tabs: Controls item
+// ---
+//
+// Items inside controls
+//
+// Notes:
+//
+// 1. Set width for button min width
+//
+
+.c-tabs__controls-item {
+    display: flex;
+    flex: 1 0 $tabs__item-size; // 1
+
+    text-align: center;
+
+    &.c--current {
+
+        .c-tabs__button {
```

Comment:
Maybe it's not as bad to couple related sub-components together (compared to coupling entirely separate components together) but I wonder if this could be simplified a bit.

Perhaps instead of setting background colors on the `c-tabs__button`, the background colors could be applied to `c-tabs__controls-item`. This would help get rid a bit of high selector specificity.

```
.c-tabs__controls-item {
    // ...

    &.c--current {
        background: $tabs__active-background;
    }
}

.c-tabs__button {
    // ...
    background: transparent;
}

```